_posts/fr/newsletters/2023-10-25-newsletter.md

```diff
@@ -0,0 +1,427 @@
+         ses remplacements de la dépense de Bob, ce qui signifie que Bob peut réduire davantage la rentabilité de l'attaque si Mallory
+         réussit. Considérez notre formule approximative `x*y*z` décrite dans la section _atténuation du rebroadcasting_. Si le coût
+         de _x_ est augmenté pour certaines des rediffusions, le coût global pour l'attaquant augmente et la valeur maximale sûre
+         d'un HTLC est plus élevée.
+
+         Riard soutient dans son document que les coûts peuvent ne pas être symétriques, en particulier pendant les périodes où les
+         frais typiques augmentent et où l'attaquant peut parvenir à faire expulser certaines de ses transactions des pools de
+         mémoire des mineurs. Sur la liste de diffusion, il [soutient également][riard cycle2] qu'un attaquant peut étendre son
+         attaque à plusieurs victimes en utilisant une forme de [regroupement des paiements][topic payment batching], augmentant
+         légèrement son efficacité.
+
+         Matt Corallo [remarque][corallo cycle2] le principal inconvénient de cette approche par rapport au simple rebroadcasting :
+         même si Bob bat l'attaquant, Bob perd une partie de la valeur HTLC (ou potentiellement la totalité). Théoriquement, un
+         attaquant ne défierait pas un défenseur s'il pense qu'il pratique une politique de destruction mutuelle, donc Bob n'aurait
+         jamais réellement besoin de payer des frais de plus en plus élevés.
+         En pratique ça n'a pas encore été prouvé sur le réseau Bitcoin.
+
+     - **Nouvel essai automatique des transactions passées** : Corallo [a suggéré][corallo cycle1] que "la seule solution à ce problème
+         sera lorsque les mineurs conserveront un historique des transactions qu'ils ont vues et les réessayeront après [...] une
+         attaque de ce type". Bastien Teinturier [a répondu][teinturier cycle] : "Je suis d'accord avec Matt, cependant, il est
+         probable qu'un travail plus fondamental doive être effectué au niveau de la couche bitcoin pour permettre aux protocoles
+         L2 d'être plus robustes contre cette classe d'attaques." Riard a également déclaré quelque chose de similaire, "une solution
+         durable ne peut se produire qu'au niveau de la couche de base, par exemple en ajoutant un historique intensif de toutes
+         les transactions observées".
+
+     - **Augmentation des frais pré-signés:** Peter Todd a soutenu que, "la bonne façon de faire des transactions pré-signées est de
+         pré-signer suffisamment de transactions *différentes* pour couvrir tous les besoins raisonnables d'augmentation des frais.
+         [...] Il n'y a aucune raison pour que les transactions B->C restent bloquées." (Emphase dans l'original.)
+
+             Cela pourrait fonctionner comme ceci : pour l'HTLC entre Bob et MalloryB, Bob donne à MalloryB dix signatures différentes
```

Comment:
Formatting issue here, it is doing a blockquote. Potentially too many indentations?